@elseym when bumping `wireguard-tools`, the kernel module will be affected as well as they share the same source (see `pkgs/os-specific/linux/wireguard/default.nix` for further reference).

I tried to fix the breaking build with the following patch locally:

``` diff
diff --git a/pkgs/os-specific/linux/wireguard/default.nix b/pkgs/os-specific/linux/wireguard/default.nix
index e103de3e1b7..e13828d018a 100644
--- a/pkgs/os-specific/linux/wireguard/default.nix
+++ b/pkgs/os-specific/linux/wireguard/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, kernel, wireguard-tools, perl }:
+{ stdenv, kernel, wireguard-tools, perl, libmnl }:
 
 # module requires Linux >= 3.10 https://www.wireguard.io/install/#kernel-requirements
 assert stdenv.lib.versionAtLeast kernel.version "3.10";
@@ -21,6 +21,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ perl ] ++ kernel.moduleBuildDependencies;
 
+  buildInputs = [ libmnl ];
+
   buildPhase = "make module";
 
   meta = with stdenv.lib; {
```

However it breaks with the following error then:

```
CC      /build/source/src/tools/config.o
  LD      /build/source/src/tools/wg
install: cannot create directory '/usr': Permission denied
make[1]: *** [Makefile:76: install] Error 1
make: *** [Makefile:62: tools-install] Error 2
builder for '/nix/store/r9qpzhaw823c2jq2pz0a7fdv5g2a5956-wireguard-0.0.20190227.drv' failed with exit code 2
error: build of '/nix/store/r9qpzhaw823c2jq2pz0a7fdv5g2a5956-wireguard-0.0.20190227.drv' on 'ssh://builder' failed: builder for '/nix/store/r9qpzhaw823c2jq2pz0a7fdv5g2a5956-wireguard-0.0.20190227.drv' failed with exit code 2
builder for '/nix/store/r9qpzhaw823c2jq2pz0a7fdv5g2a5956-wireguard-0.0.20190227.drv' failed with exit code 1
error: build of '/nix/store/r9qpzhaw823c2jq2pz0a7fdv5g2a5956-wireguard-0.0.20190227.drv' failed
```

This is probaly related to some `make` script missing the proper install prefix. Unless you're faster, I'll have another look at it this weekend :)

--- a/pkgs/os-specific/linux/wireguard/default.nix
+++ b/pkgs/os-specific/linux/wireguard/default.nix
@@ -21,7 +21,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ perl ] ++ kernel.moduleBuildDependencies;
 
-  buildPhase = "make module";
+  buildFlags = [ "module" ];
+  installTargets = [ "module-install" ];
 
   meta = with stdenv.lib; {
     inherit (wireguard-tools.meta) homepage license maintainers;

